projenrc/workflow-no-docker-patch.ts

```diff
@@ -22,7 +22,7 @@ export class WorkflowNoDockerPatch {
       JsonPatch.add(`/jobs/${workflow}/steps/`, {
         name: 'Setup Node.js',
         uses: 'actions/setup-node@v3',
-        with: { 'node-version': project.minNodeVersion ?? '14.x' },
+        with: { 'node-version': project.minNodeVersion ?? '16.x' },
```

Comment:
Curious that since Node 16 has reached end of life on September 11, 2023, any particular reason not using a new version of Node?

Reply:
See above comment